grugthink/docker/Dockerfile.base

```
@@ -0,0 +1,33 @@
+# GrugThink Base Image - Heavy ML Dependencies
+FROM python:3.11-slim
+
+WORKDIR /app
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)